cfme/tests/infrastructure/test_timelines.py

```diff
@@ -105,6 +157,9 @@ def _power_on(self):
     def _power_off(self):
         return self.vm.provider.mgmt.stop_vm(self.vm.name)
 
+    def _power_off_power_on(self):
```

Comment:
Question: this should be vm restart?

Reply:
I wanted to make clear that it will not be calling the provider.mgmt.restart_vm() method, to differentiate it with the others, used to generate events. but maybe it is useless. i will rename it.